resources/function_help/json/array_minority

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "array_minority",
+  "type": "function",
+  "groups": ["Arrays"],
+  "description": "Return the less common values in an array.",
+  "arguments": [
+    {"arg":"array","description":"an array"},
+    {"arg":"option='all'", "optional":true, "description":"a string specifying the return values handling. Valid options are:<br /><ul><li>all: Default, all less common values are returned in an array</li><li>first: Return the first of the less common values</li><li>median: Return the median of the less common values</li><li>real_minority: Return all values except the real majority if thre is one</li></ul>"}
+  ],
+  "examples": [
+    { "expression":"array_minority(array(0,42,42), 'all')", "returns":"[ 0 ]"},
+    { "expression":"array_minority(array(0,1,42,42), 'all')", "returns":"[ 0, 1 ]"},
+    { "expression":"array_minority(array(0,1,42,42,43,1), 'first')", "returns":"43"},
+    { "expression":"array_minority(array(1,2,3,3), 'median')", "returns":"1.5"},
+    { "expression":"array_minority(array(0,1,42,42,43), 'real_minority')", "returns":"[ 42, 43, 0, 1 ]"},
```

Comment:
In the output, should we keep the order values appear in the input, ie [ 0, 1, 42, 43 ]? Someone may want to iterate over the output and do an action based on their position. And I think it's currently "unpredictable".

Reply:
This functions are implemented using a QHash, which provides faster key look up than a QMap at the cost of unpredictable order (for the 'first' parameter is the same). [https://doc.qt.io/qt-5/qhash.html#details](https://doc.qt.io/qt-5/qhash.html#details)
If predictable output sort order is preferred over speed I could replace QHash with the QMap, is a small change.

With the current implementation you can still use the function array_prioritize (see pr #41338) to reorder the output like you described:
`array_prioritize(array_minority(array(0,1,42,42,43), 'real_minority'), (array(0,1,42,42,43)) -> [ 0, 1, 42, 43 ]`